templates/CRM/Contact/Page/DedupeException.tpl

```diff
@@ -27,55 +27,120 @@
   {ts}Filter Contacts{/ts}
 </div>
 <div class="crm-accordion-body">
-  <table class="no-border form-layout-compressed" id="searchOptions" style="width:100%;">
-    <tr>
-      <td class="crm-contact-form-block-contact1">
-        <label for="contact1">{ts}Contact 1{/ts}</label><br />
-        <input class="crm-form-text" type="text" placeholder="Search Contact1" search-column="0" />
-      </td>
-      <td class="crm-contact-form-block-contact2">
-        <label for="contact2">{ts}Contact 2{/ts}</label><br />
-        <input class="crm-form-text" type="text" placeholder="Search Contact2" search-column="1" />
-      </td>
-    </tr>
-  </table>
+  <form method="get">
+    <table class="no-border form-layout-compressed" id="searchOptions" style="width:100%;">
+      <tr>
+        <td class="crm-contact-form-block-contact1">
+          <label for="search-contact1">{ts}Contact Name{/ts}</label><br />
+          <input type="text" size="50" placeholder="Search Contacts" value="{$searchcontact1}" id="search-contact1" search-column="0" />
+        </td>
+        <td class="crm-contact-form-block-search">
+          <label>&nbsp;</label><br />
+          <button type="submit" class="button crm-button filtercontacts"><span><i class="crm-i fa-search"></i> Find Contacts</span></button>
+        </td>
+      </tr>
+    </table>
+  </form>
 </div>
-<table id="dedupeExceptions" class="display crm-sortable" data-searching='true' data=filter='false'>
-  <thead>
-    <tr class="columnheader">
-      <th>{ts}Contact 1{/ts}</th>
-      <th>{ts}Contact 2 (Duplicate){/ts}</th>
-      <th data-orderable="false"></th>
-    </tr>
-  </thead>
-  <tbody>
-    {foreach from=$dedupeExceptions item=exception key=id}
-      <tr id="dupeRow_{$id}" class="{cycle values="odd-row,even-row"}">
-        <td><a href ={crmURL p='civicrm/contact/view' q="reset=1&cid=`$exception.main.id`"}>{$exception.main.name}</a></td>
-        <td><a href ={crmURL p='civicrm/contact/view' q="reset=1&cid=`$exception.other.id`"}>{$exception.other.name}</a></td>
-        <td><a id='duplicateContacts' href="#" title={ts}Remove Exception{/ts} onClick="processDupes( {$exception.main.id}, {$exception.other.id}, 'nondupe-dupe', 'dedupe-exception' );return false;">&raquo; {ts}Remove Exception{/ts}</a></td>
```

Comment:
@agileware Just to note i think we need to re-add this as well as the 3rd <th> otherwise of the main reasons for the page is taken away

Reply:
@seamuslee001 please ping me - @agileware is a github team and no one is notified.